src/library_exceptions.js

```diff
@@ -194,16 +192,18 @@ var LibraryExceptions = {
   },
 
   __cxa_uncaught_exceptions__deps: ['$uncaughtExceptionCount'],
-  __cxa_uncaught_exceptions: function() {
-    return uncaughtExceptionCount;
-  },
+  __cxa_uncaught_exceptions: () => uncaughtExceptionCount,
 
-  __cxa_call_unexpected: function(exception) {
-    abort('Unexpected exception thrown, this is not properly supported - aborting');
+  __cxa_call_unexpected: (exception) => {
+    err('Unexpected exception thrown, this is not properly supported - aborting');
+#if !MINIMAL_RUNTIME
+    ABORT = true;
+#endif
+    throw exception;
```

Comment:
why did this change from calling `abort`? The previous code was shorter and more obvious to me...

Reply:
Oh strange.. this must be part of some other change.

I guess maybe it was so that the original exception was not lost, but I actually don't remember making this change :)